deploy-chroot/enter-internal.sh

```diff
@@ -3,7 +3,9 @@
 set -euo pipefail
 
 sudo mount -o rw,remount /proc/sys
-sudo mknod --mode='ug=rw,o=' /dev/kvm c 10 "$(grep -w kvm /proc/misc | cut -f 1 -d ' ')"
+if grep -wq kvm /proc/misc; then
+    sudo mknod --mode='ug=rw,o=' /dev/kvm c 10 "$(grep -w kvm /proc/misc | cut -f 1 -d ' ')"
+fi
 sudo chgrp kvm /dev/kvm
```

Comment:
Maybe stick the chgrp in the if?
Might also be worth explicitly warning that it couldn't create the device node?

Reply:
wooops